analyzers/tests/SonarAnalyzer.Test/Rules/LoggersShouldBeNamedForEnclosingTypeTest.cs

```diff
@@ -31,7 +31,7 @@ public class LoggersShouldBeNamedForEnclosingTypeTest
     public void LoggersShouldBeNamedForEnclosingType_CS() =>
         Builder
             .AddPaths("LoggersShouldBeNamedForEnclosingType.cs")
-            .AddReferences(NuGetMetadataReference.MicrosoftExtensionsLoggingAbstractions())
+            .AddReferences(NuGetMetadataReference.MicrosoftExtensionsLoggingPackages(Constants.NuGetLatestVersion))
```

Comment:
Constants.NuGetLatestVersion is the default value for this parameter. No need to specify it.

Reply:
This one does not have yet a default parameter. Only the abstractions.